docs/admin/configuration/radicale-integration.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 12
+id: radicale-integration
+title: Calendar and Contacts Integration with Radicale
+description: "How to enable Calendar and Contacts integration via Radicale in OpenCloud."
+---
+
+OpenCloud is able to act as an authenticating reverse proxy for the open-source
+CalDAV and CardDAV Server [Radicale](https://radicale.org).
+
+The `opencloud_full` deployment example already contains all the required pieces to
+setup the integration. This guide explains the required steps to achieve that.
+
+# Disclaimer
+
+⚠️ Please note ⚠️
```

Comment:
You could/should use the Admonitions coming with Docusaurus to sign out a note: https://docusaurus.io/docs/next/markdown-features/admonitions

Basically, you just need to surround the paragraph with 
```
:::note
A note for the admins!
:::
```
and get the note nicely rendered.

Reply:
Nice. Thank you.